apollo-runtime/src/main/java/com/apollostack/android/GraphqlService.java

```diff
@@ -0,0 +1,14 @@
+package com.apollostack.android;
+
+import android.database.Observable;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/** A {@link retrofit2.Retrofit} service for executing GraphQL requests. */
+public interface GraphqlService {
```

Comment:
Would be nice if exclude this settings at all, as it increases friction of adoption this library, we already have big one: node.js. I guess we don't want to user to have a big doc how to set-up all these stuff